### DISCUSSION TITLE
Appium integration

### DISCUSSION THREAD

**Discussion by lozik4** [DeviceFarmer/stf#790] [Q&A] [answered]
Please tell me if it is possible to run Appium tests using this interface. 
I mean, can I install the application using the Appium client, test the mobile application, and then uninstall this application?
Also, did I understand correctly that to use iOS I need to deploy this service on MacOS?

**Comment by jupe** [comment, marked as answer]
Sure, appium usea ADB with android devices and stf provide ADB for client. Here’s client written with python that works well with appium and stf: https://github.com/OpenTMI/stf-appium-python-client . There’s also pytest plugin top of that. With iOS, you are more or less alone since there is no official ios support at stf…
